terra/views.py

```diff
@@ -678,12 +680,14 @@ def render_to_response(self, context, **response_kwargs):
                                 [
                                     v["subunit"].manager,
                                     actualexpense.treq.traveler.unit,
-                                    actualexpense.treq.traveler.uid,
+                                    actualexpense.treq.traveler.unit.manager,
+                                    str(actualexpense.treq.traveler.uid),
```

Comment:
@aprigge Why did this `traveler.uid` in line 684 get newly wrapped in `str()`, while the value in line 704 did not?  Should they be consistent?  Is the `str()` necessary?

Reply:
Thanks, updated now